mono/metadata/boehm-gc.c

```diff
@@ -109,7 +109,7 @@ void
 mono_gc_base_init (void)
 {
 	MonoThreadInfoCallbacks cb;
-	const char *env;
+    char *env;
```

Comment:
@alexanderkyte `env` is later used for usage around `g_getenv` and later `g_free` is used;  is that correct?